src/framework/mlt_property.c

```diff
@@ -937,7 +938,7 @@ void mlt_property_pass( mlt_property self, mlt_property that )
 
 	if ( self->types & mlt_prop_int64 )
 		self->prop_int64 = that->prop_int64;
-	else if ( self->types & mlt_prop_int )
+	else if ( self->types & mlt_prop_int || self->types & mlt_prop_color ) // warning
```

Comment:
I also wonder why the warning is needed here.

Reply:
See above

Reply:
Did you intend to remove this "//warning"?

Reply:
removed